demo/src/App.scss

```diff
@@ -102,3 +115,14 @@ body {
     color: #878787;
   }
 }
+
+.ant-checkbox-wrapper:hover .ant-checkbox-inner,
+.ant-checkbox:hover .ant-checkbox-inner,
+.ant-checkbox-input:focus + .ant-checkbox-inner {
+  box-shadow: $focus-shadow;
+}
+
+.css-vbks9d {
+  border: 2px solid #157dc6;
+  box-shadow: $focus-shadow;
+}
```

Comment:
I believe you added this to make the outline on the dropdown the same as what you added for the links and checkboxes correct? I would prefer this not to be added since the dropdown does already have an indicator and this makes the dropdown outline look different than it would be if people used it out of the box.

Reply:
I added for better clarity, since originally only indication on the border doesn't have enough contrast and pop.

Reply:
That's a fair point. Perhaps something that I should consider changing at the component level. I might change this with the style refactors I am working on currently.